Makefile.toml

```diff
@@ -3,6 +3,7 @@
 [env]
 CARGO_MAKE_EXTEND_WORKSPACE_MAKEFILE = true
 
+# linting
```

Comment:
no need to change on this PR, but for the future, these comments that only tag code are not necessary. This is repeating the context we have in the `.toml` itself.

Reply:
This is needed by cargo-make to enable the "workspace" mode, if this flag is not set cargo-make does not work properly.

Reply:
ah, thanks for the explanation, i didn't know that. Can you please give links to the docs about this?

Reply:
Sure, I will add them to the code.